adhoc_customize/CarMsg msg
---
bool success
string[] failed_dst
